packages/browser/src/tracing/request.ts

```diff
@@ -374,6 +376,7 @@ export function xhrCallback(
             'server.address': host,
             [SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN]: 'auto.http.browser',
             [SEMANTIC_ATTRIBUTE_SENTRY_OP]: 'http.client',
+            body: requestBody,
```

Comment:
h: We should not do this - this will attach the request bodies to all spans, always, which is a) large, b) potentially PII sensitive 😬 

Instead of doing this in `on('spanStart')`, I think we'll need a hook that provides the request or body in some way. I am thinking of a new hook like:

```js
client.on('outgoingRequestSpanStart', (span: Span, { body }: { body: unknown }) => {
  // ...
});
```

and emit this hook in this file after the span was started 🤔

Reply:
Noted. I implemented that hook and attached the req payload only to graphql spans.